tools/ci_build/build.py

```diff
@@ -328,6 +341,9 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
 
     cmake_args += ["-D{}".format(define) for define in cmake_extra_defines]
 
+    toolchain_path = os.path.join('cmake', 'external', 'vcpkg', 'scripts', 'buildsystems', 'vcpkg.cmake')
+    cmake_args += ["-DCMAKE_TOOLCHAIN_FILE=" + toolchain_path]
```

Comment:
It is OK for now. But if any CMakeLists.txt using `find_package()` and vcpkg happens to have that package, then the build end up using the vcpkg package other than the package in the repo. You could set the `CMAKE_TOOLCHAIN_FILE` in the hosting.cmake file with an environment variable. In this file you just set the environment variable.

Reply:
Done